src/plugins/data/common/index_patterns/fields/types.ts

```diff
@@ -9,8 +9,8 @@ import { IndexPatternFieldBase } from '@kbn/es-query';
 import { FieldSpec, IndexPattern } from '../..';
 
 /**
- * @deprecated
- * Use IndexPatternField or FieldSpec instead
+ * @deprecated Use `IndexPatternField` or `FieldSpec` instead
```

Comment:
I think if to use `{@link IndexPatternField}` in such cases, then it will nicely popup as a link in docs